Webhook Lambda Handler Documentation

benbpyle opened the pull request:
Added in the README and images to support

benbpyle committed:
Webhook Lambda Handler Documentation

Added in the README and images to support

allenheltondev commented on docs/topics/webhooks/lambda-handler.md:
```diff
@@ -0,0 +1,116 @@
+## Deploying an AWS Lambda Webhook Handler
```
This should be an h1, meaning only one `#` sign. Let's also rename this to 
"Configure Momento Topics to publish events to Amazon EventBridge"

allenheltondev commented on docs/topics/webhooks/lambda-handler.md:
````diff
@@ -0,0 +1,116 @@
+}
+```
+
+### Event Structure
````
Our style guide only capitalizes the first word in a heading, can you please go through and update capitalization. 
The exception to the rule is proper nouns/products and code/properties like you have below.

allenheltondev commented on docs/topics/webhooks/lambda-handler.md:
```diff
@@ -0,0 +1,116 @@
+
+The message detail will contain the actual payload as supplied from the webhook. By not removing elements from the message, maximum detail is provided to all consuming applications. For reference, the Momento webhook payload fields are [described here.](https://docs.momentohq.com/topics/webhooks/overview#example-event)
+
+## Notes and Wrapping Up
```
To make it feel less bloggy/story, Maybe change this to "Considerations" and remove the storytelling part of the first paragraph in here.

allenheltondev commented:
Once [this PR](https://github.com/momentohq/public-dev-docs/pull/657) is merged, please move this file under the integrations folder at the root of the Topics page.

Also, we need the frontmatter I sent you in DM added to the documentation as well. I know you know that but putting it here for reference.

benbpyle committed:
Merge branch 'momentohq:main' into lambda-webhook-handler-doc

benbpyle committed:
Changes re PR Notes

1. Moved files under top-level integrations directory
2. Added in the intro content that matters for the docs web structure
3. Modified initial heading content per PR request
4. Changed top header to by H1 from H2
5. Removed capitalization from words that aren't proper in title headings and aren't the first word in the heading
6. Changed "Notes" to "Considerations" and rewrote the first paragraph in that section

benbpyle commented:
@allenheltondev  modified the source to match the PR notes.  Description of changes is in the commit message description

benbpyle committed:
Update intro information

benbpyle committed:
Update documentation for one-click and note

Added the Admonitions
Added the URL for one-click

benbpyle committed:
Target _blank added

benbpyle committed:
Update image locations and references

benbpyle committed:
One-click image path

benbpyle committed:
Adjust for Docusaurus build
